src/components/tag/_tag.scss

```diff
@@ -7,7 +7,13 @@
     @include govuk-font($size: 16, $weight: bold, $line-height: 1.25);
 
     display: inline-block;
-    padding: govuk-spacing(1) 8px 0;
+    padding: 4px 8px;
+    // Since New Transport sits slightly higher than other common fonts.
+    // We use intentionally uneven padding to make it balanced, this can be
+    // removed using the version of the font that has a more common vertical spacing.
+    @if $govuk-font-family == $govuk-font-family-nta {
```

Comment:
If this isn't true, then the alpha tag ends up with 4px on top and bottom, which makes it 28px – considerably larger than the 'with nta' variant and no longer following our /5 rhythm. I think if we're going to try and make this 'conditional' we should make sure that the component still follows our rules in both cases.

In other places we're just commenting and applying adjustments by taking from one side and adding to the other, e.g. like this: https://github.com/alphagov/govuk-frontend/blob/master/src/components/button/_button.scss#L164-L178

Reply:
My thought process:

1. As far as I know the 5 pixels is a scale based on how NTA works (might be wrong)
2. The component being a little bit bigger is much better than it looking completely broken as seen in the screenshots.

I'll make this less clever and we'll figure this out when we do proper theming support.

Reply:
The 5px thing is not related to the font, it's a decision based on the font-size / line-height combination. Each font size has a line-height rounded to the nearest 5px increment. This is why the spacing scale is increments of 5px and 10px and the inputs, buttons etc have been standardised at 40px (45px with the start button)

I say this, but this has mainly been considered for the main content of the page, there were way too many variables to make it work properly for the header and beforeContent area...haven't checked it in a while.